kie-wb-parent/kie-wb-webapp/src/main/resources/security-module-process-admin.properties

```diff
@@ -19,6 +19,7 @@ role.process-admin.permission.perspective.read.UserHomePagePerspective=true
 role.process-admin.permission.perspective.read.ProcessDefinitions=true
 role.process-admin.permission.perspective.read.ProcessInstances=true
 role.process-admin.permission.perspective.read.Tasks=true
+role.process-admin.permission.perspective.read.TaskAdmin=true
```

Comment:
What are the roles that should have access to Task Aministration perspective?
I would expect only `admin` and `process-admin`. However when tested with other roles, I could also navigate to this perspective with user with just `analyst` or `developer` roles..
![screenshot from 2017-05-05 08-10-18](https://cloud.githubusercontent.com/assets/2716069/25735269/683be650-316a-11e7-9055-9c4f2afd50e6.png)